mora/auth/base.py

```diff
@@ -89,25 +104,19 @@ def login():
         raise exceptions.HTTPException(
             exceptions.ErrorCodes.E_CONNECTION_FAILED)
 
-    resp = flask.jsonify({
-        "user": username,
-    })
+    flask.session['username'] = username
 
     if assertion:
-        resp.set_cookie(
-            COOKIE_NAME, assertion,
-            secure=flask.request.is_secure,
-            httponly=True,
-        )
+        flask.session[TOKEN_KEY] = assertion
     else:
-        resp.delete_cookie(COOKIE_NAME)
+        flask.session.clear()
 
-    return resp
+    return flask.make_response()
```

Comment:
I was about to complain about this, but actually, it's vaguely more clear than a mere:

```python
    return ''
```